### PR TITLE
Added Homepage + page routes

### PR DESCRIPTION
**Pull request by nayyyhaa**
## In this PR :
-  I've just added **Homepage + its components**
-   Other page routes are integrated but haven't integrated their contents yet.
- Have added data for homepage

### Files to check
- [x] src/App.js

#### Component files

- [x] components/[Card.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-7916963985f6b54523951aa9bdcdb855db5905bed40fc7775a485986aec0fa5d)
- [x] components/[FeatureCategories.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-ca30510ab0c5aa9bde117caa4700116fff90df316cfd661598130e8daa6556df)
- [x] components/[Footer.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-bbc61c7a463361cf5c362945de11c994e0f9d8d0355858d6c0323432bdba3053)
- [x] components/[HeaderAnnouncement.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-bc22f1fab5afda9950d704f98cb465a74e38af0ecda65ad3e76c1faa223ec925)
- [x] components/[Marquee.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-eb4a556f3804cbb343d32d8996dbaf0d808d329ee9cfb31d0c44f528d5677551)
- [x] components/[Navbar.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-c2bbe5525aa57a971df65b233f8bbe05d6861f64b49bb9d5e0475a3d6ec44838)
- [x] components/[ProductsListing.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-c08da705d96013060bd9df0c0c7c7c70bdd2d34323df49be71f3b9d5f27f355e)
- [x] components/[SiteHeader.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-9f855bffec99553afc5a1889bbf526ddcb41c8392d6774dc28627ca6363e48f5)
- [x] components/[index.js](https://github.com/nayyyhaa/inNotion-care-react/commit/b4b170b55bd919f606983825bd41bbfa01877ead#diff-c7db7feaa21621735b0b41422b0d1ba0aae29948113c1a3bf63575d63217b10a)

#### Page file

 - [x] [src/pages/Homepage.js](https://github.com/nayyyhaa/inNotion-care-react/commit/3d5e632cb6b6631fa733b7662c475f735e3123a7#diff-1063f1bef92d5c0fee5f64413fed09584b31b2d9ab0873a1166a7a2f8294aa4c)
 
#### files to *ignore*
- [ ] CSS files
- [ ] data files
- [ ] other pages (they are blank for now, added them just for setting up routes)
- [ ] package.json

### Live preview : https://deploy-preview-1--innotion-carev2.netlify.app/

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        </ul>
+        <label className="field searchfield w-20p" htmlFor="search-text">
+          <span className="search-icon cursor p-h-1">
+            <i className="fa fa-search" aria-hidden="true"></i>
+          </span>
+          <input type="search" className="input search-nav reset-ip p-05" placeholder="Search here" id="search-text" />
+        </label>
+        <div className="nav-icon-btns row-flex no-wrap">
+          <ul className="row-flex no-bullet">
+            <li className="nav-icon-btn icon-btn rd-bdr grid-ctr wt-text m-r-3">
```
Comment (hsnice16) [inline src/components/Navbar.js]: Here, these Lists are repeating. Maybe you can think to loop over an array and then create these lists.

File: src/components/Marquee.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { MarqueeData } from "../data";
+
+export const Marquee = () => {
+  return (
+    <div className="marquee p-2">
+      <div className="marquee-container">
+        {MarqueeData?.map(({ id, title, image }) => {
```
Comment (hsnice16) [inline src/components/Marquee.js:7]: This is nice, you are destructuring the properties and then using them.

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                  required=""
+                />
+                <button className="btn secondary-outline-btn" type="submit">
+                  Submit
+                </button>
+              </label>
+            </div>
+          </form>
+          <div className="contact-socials left-text full-wd m-v-1">
+            <a className="icon-btn footer-icon grey-text p-1" href="mailto:nehaguptang.ng@gmail.com">
```
Comment (hsnice16) [inline src/components/Footer.js]: Here too, these links you can create by looping over an array.

**Review by hsnice16 — APPROVED** [review]
Code quality is superb. Just have added a few suggestions.

**Review by BalajiHarinath — APPROVED** [review]
Suggestion : As Himanshu suggested you can map all the repetitive blocks.

Appreciation: Folder structure is good, clean UI and I liked the scrolling animation which you have used.  Good that you have used css animation rather than using marquee html tag because some browsers are no longer supporting it.

File: src/components/Navbar.js (hunk 1 above)
Comment (nayyyhaa) [reply src/components/Navbar.js]: Done with the changes. I've added Navbar & footerData for looping.

**Comment by nayyyhaa** [comment]
Thanks @hsnice16 & @BalajiHarinath for reviewing. I've made the said changes, now merging the PR.

**Pull request merged**